tests/integration_tests/style/test_rust.py

```diff
@@ -7,6 +7,19 @@
 from framework import utils
 
 
+def test_rust_order(monkeypatch):
+    """
+    Tests that `Cargo.toml` dependencies are alphabetically ordered.
+
+    @type: style
+    """
+
+    # Re-direct to repository root.
+    monkeypatch.chdir("..")
```

Comment:
nit: this could be included in the below command as `cwd=".."`

Reply:
Fixed.